akshathjain/sliding_up_panel
# fix unscrollable lists when slide direction is down. handle reverse lists.

Fixes #215 

This PR fixes the following scenarios

- Scrollable content in the panel when `slideDirection == SlideDirection.DOWN` did not scroll at all
- Strange list behavior when `reverse == true` for both `SlideDirection.DOWN` and `SlideDirection.UP`

To test

1. Run the example project
2. Use all 4 of the following configurations and verify that each one opens, scrolls, and closes as expected.

    - List `reverse: false`, SlidingUpPanel `slideDirection: SlideDirection:UP`
    - List `reverse: true`, SlidingUpPanel `slideDirection: SlideDirection:UP`
    - List `reverse: false`, SlidingUpPanel `slideDirection: SlideDirection:DOWN`
    - List `reverse: true`, SlidingUpPanel `slideDirection: SlideDirection:DOWN`
